src/browser/actions.js

```diff
@@ -62,6 +104,16 @@ define((require, exports, module) => {
     localStorage[`session@${version}`] = JSON.stringify(session.toJSON());
   };
 
+  const showDashboard = (dashboardCursor, tabStripCursor) => {
```

Comment:
It also I guess depends what we want behavior to be, should hiding dashboard actually trigger hiding of a tabStrip ?